src/commands/junit/api.ts

```diff
@@ -36,26 +36,30 @@ export const uploadJUnitXML = (request: (args: AxiosRequestConfig) => AxiosPromi
     fileName = 'default_file_name'
   }
 
-  const spanTags: Record<string, string | undefined> = {
+  const metadata: Record<string, any> = {
```

Comment:
why `any`? Could we scope it down? Like `string | boolean | undefined | {[key:string]: string }`?

Reply:
This matches how the backend handles it now. Any type is allowed in the value really so I'm not sure I see the point but  I don't know typescript so if you think it's still better we can change it

Reply:
yeah ok, let's leave it as is